src/main/java/com/minecraftabnormals/upgrade_aquatic/core/mixin/DrownedRendererMixin.java

```diff
@@ -0,0 +1,44 @@
+package com.minecraftabnormals.upgrade_aquatic.core.mixin;
+
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Overwrite;
+
+import com.mojang.blaze3d.matrix.MatrixStack;
+
+import net.minecraft.client.renderer.entity.AbstractZombieRenderer;
+import net.minecraft.client.renderer.entity.DrownedRenderer;
+import net.minecraft.client.renderer.entity.EntityRendererManager;
+import net.minecraft.client.renderer.entity.model.DrownedModel;
+import net.minecraft.entity.monster.DrownedEntity;
+import net.minecraft.util.math.MathHelper;
+import net.minecraft.util.math.vector.Vector3f;
+
+@Mixin(DrownedRenderer.class)
+public class DrownedRendererMixin extends AbstractZombieRenderer<DrownedEntity, DrownedModel<DrownedEntity>> 
```

Comment:
This class has many problems in general. There's an abundance of formatting errors and unnamed local variables in the methods. 
Another thing is you decided to use a different bracket-style only for this class?